### DISCUSSION TITLE
Washer disappeared from Homebridge

### DISCUSSION THREAD

**Discussion by urbanhaas** [thoukydides/homebridge-homeconnect#111] [General]
My Bosch Washer was in Homebridge, but has disappeared. I'm on the latest version.

Is there some change on the API side? Is there a way to turn on debugging?

**Comment by thoukydides** [comment]
There are no advertised API changes that would account for that:
https://developer.home-connect.com/changelog

My own Siemens iQ700 Dishwasher (SN678D06TG/53) still appears in HomeKit via this plugin and is fully functional.

Hence, I suspect that the problem is with your appliance and its connection to the Home Connect servers.

Does it appear and work in the Home Connect app?
How about if you turn off Wi-Fi on your phone, so that it has to go via the Home Connect servers rather than communicating directly with the dishwasher?
Within the Home Connect app go to the appliance's Settings, and scroll down to the Network section. Does it show three green lines?
Have you tried switching the dishwasher off at the mains for a few seconds and then back on again?

This plugin writes extensive debug information to the Homebridge log. If you run Homebridge in debug mode, by adding the `-D` command line option, then it also logs every Home Connect API call and its result.

**Reply by urbanhaas** [reply]
I also have a dryer and it still appears in HomeBridge.

Yes, it's still in HomeConnect app, but the settings are blank (compared to the dryer which actually shows the settings). It looks like I will have to delete it in the app and re-connect it in there.

Thanks for the -D (deubg) option. I'll go that route if this delete/add doesn't fix it. HomeConnect has had issues like this in the past, so looks like it's an issue there.

**Reply by urbanhaas** [reply]
It was horked in the Homebridge connection. All fixed. Thanks for the debug tips.
